Handling Devices with Firmware >= 5.8 but no humidity and pressure sensor

pmneo opened the pull request:
Added Debug Output
Handling Devices with Firmware >= 5.8 but no humidity and pressure sensor
Fixed Sensor Tab Mapping

knro commented on indi-aagcloudwatcher-ng/CloudWatcherController_ng.cpp:
```diff
@@ -143,34 +143,38 @@ bool CloudWatcherController::getAllData(CloudWatcherData *cwd)
 
         if (!check)
         {
+            LOG_DEBUG( "ERROR in getIRSkyTemperature" );
```
LOG_ERROR

knro commented on indi-aagcloudwatcher-ng/CloudWatcherController_ng.cpp:
```diff
@@ -143,34 +143,38 @@ bool CloudWatcherController::getAllData(CloudWatcherData *cwd)
 
         if (!check)
         {
+            LOG_DEBUG( "ERROR in getIRSensorTemperature" );
```
Same as above

knro reviewed (changes requested):
Use LOG_ERROR or LOGF_ERROR to report errors. Please update as well Debian changelog.

pmneo commented:
Is this correct?

indi-aagcloudwatcher-ng (1.7) focal; urgency=medium

  * Added Debug Output
  * Handling Devices with Firmware >= 5.8 but no humidity and pressure sensor
  * Fixed Sensor Tab Mapping

 -- Philip Mair  <philip@pmneo.de>  Sun, 7 August 2022 12:00:00 +0200

CS
